README.md

```diff
@@ -148,8 +148,7 @@ They will be populated with IDs that reference the new derived tables.
                                    csvcol:dbcol(TYPE),...
       --filename-column TEXT       Add a column with this name and populate with
                                    CSV file name
-      --fixed-column TEXT          For colname:value, add a column colname and
-                                   populate with value
+      --fixed-column TEXT          Populate column with a fixed value
```

Comment:
With your new code the `--help` text actually looks like this:

      --fixed-column <TEXT TEXT>...  Populate column with a fixed value

Reply:
Good catch!  Thanks.